src/main/java/org/hawkular/client/android/backend/model/MetricAvailabilityBucket.java

```diff
@@ -39,6 +39,7 @@ public MetricAvailabilityBucket createFromParcel(Parcel in) {
 
     protected MetricAvailabilityBucket(Parcel in) {
         value = in.readString();
+        empty = in.readString().equals("true");
```

Comment:
@pilhuhn I feel it fills that gap from both end. `android.os.Parcel` do not have real and write method for boolean so forced to save in string or any other form.